src/Professions/Professions.java

```diff
@@ -0,0 +1,62 @@
+package Professions;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import java.util.function.BiFunction;
+
+/**
+ * Keeps track of all static data and helper methods.
+ * Responsible for static factory initialization of players, resolved by profession.
+ * @author Aya
+ *
+ */
+public class Professions {
+	public static final Set<String> ALLOWED_PROFESSION_NAMES;

```

Comment:
Set for efficient "contains" method, which is `O(1)` on a set but `O(n)` on a list.